### PR TITLE
feat: implement resource manager app

### PR DESCRIPTION
**Pull request by baurine**
## What Did

Implement the resource manager app

## Preview

https://user-images.githubusercontent.com/1284531/234152491-e4cc5558-f7fb-439d-85d3-a9e3bf8a35d7.mp4

https://user-images.githubusercontent.com/1284531/234152502-44db299f-ead4-47c6-9433-3509177f07a4.mp4

https://user-images.githubusercontent.com/1284531/236599399-87ed06b1-93d6-4815-8c52-ca12c62ebd88.mp4

### CODE REVIEW

File: ui/packages/tidb-dashboard-lib/src/apps/UserProfile/components/index.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export * from './Form.SSO'
+export * from './Form.Session'
+export * from './Form.PrometheusAddr'
+export * from './Form.Version'
```
Comment (shhdgit) [inline ui/packages/tidb-dashboard-lib/src/apps/UserProfile/components/index.ts:4]: And `export * from './Form.Language'`?

File: ui/packages/tidb-dashboard-lib/src/apps/UserProfile/index.tsx [hunk 2]
```diff
@@ -2,11 +2,13 @@ import React, { useContext } from 'react'
...
-import { PrometheusAddressForm } from './Form.PrometheusAddr'
-import { VersionForm } from './Form.Version'
-import { LanguageForm } from './Form.Language'
+import {
+  SSOForm,
+  SessionForm,
+  PrometheusAddressForm,
+  VersionForm
+} from './components'
+import { LanguageForm } from './components/Form.Language'
```
Comment (shhdgit) [inline ui/packages/tidb-dashboard-lib/src/apps/UserProfile/index.tsx:11]: How about combine these components

**Review by shhdgit — APPROVED** [review]
The rest LGTM.

**Comment by HuSharp** [comment]
Maybe can update the video which shown in the pr description?

**Review by HuSharp — APPROVED** [review]
backend part LGTM

File: ui/packages/tidb-dashboard-lib/src/apps/UserProfile/index.tsx (hunk 2 above)
Comment (baurine) [reply ui/packages/tidb-dashboard-lib/src/apps/UserProfile/index.tsx:11]: nice catch!

**Pull request merged**